crypto/fipsmodule/ec/ec_test.cc

```diff
@@ -182,6 +182,259 @@ TEST(ECTest, Encoding) {
       y_hex.get());
 }
 
+// P-{224,256,384,521} test vectors, taken from CAVP
```

Comment:
Thanks for the precise reference

Reply:
This is actually copied from `evp_test.cc`. I think these tests should be added on `ec_test.cc`. Let me know if you want reducing the duplicates.

Reply:
Ahh I added that. I thought it looked familiar :)

I think it is fine for now.